mne/utils/check.py

```diff
@@ -147,8 +147,9 @@ def _check_event_id(event_id, events):
     return event_id
 
 
+@verbose
 def _check_fname(fname, overwrite=False, must_exist=False, name='File',
-                 need_dir=False):
+                 need_dir=False, *, verbose=True):
```

Comment:
In general all new `verbose` calls should be
```suggestion
                 need_dir=False, *, verbose=None):
```
Having `verbose=True` here will probably break some `verbose=False` / `verbose='error'` usages elsewhere in our code